test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -662,6 +705,35 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
 
     #endregion Redirect tests
 
+    #region SkipHeaderVerification Tests
+
+    It "Verifies Invoke-WebRequest default header handling with no errors" {
+        $headers = @{"If-Match" = "*"}
+        $response = ExecuteRequestWithCustomHeaders -Uri "http://localhost:8080/PowerShell?test=echo" -headers $headers
+
+        $response.Error | Should BeNullOrEmpty
```

Comment:
You're right but the end result does not change.  If the request does not complete successfully, I want to know that.  Testing the content doesn't help me diagnose that and, in fact, obfuscates it.